test/testdata/generate.sh

```diff
@@ -102,6 +102,34 @@ function scaffold_test_project {
       $kb create api --version v1 --kind Lakers --controller=true --resource=true --make=false
       $kb create webhook --version v1 --kind Lakers --defaulting --programmatic-validation
     fi
+  elif [[ $project =~ multimodule ]]; then
+    header_text 'Switching to multimodule & multigroup layout ...'
+    $kb edit --multimodule=true --multigroup=true
+
+    header_text 'Creating APIs ...'
```

Comment:
Probably we dont need all of these cases, but I tried to test all kinds of scenarios. Please let me know how much you actually want me to test here.

Reply:
We do not need to add the new if:
We just need to add the [[ project-v3-multimodule ]] and for [[ project-v4-multimodule ]]in the line: https://github.com/kubernetes-sigs/kubebuilder/blob/b9d0e8c764d8bfb2a18c4e25210b19ee186759d5/test/testdata/generate.sh#L48 (so that we will have the same scaffold done for the other mocks but using this layout )

Then, at the bottom of this file, you will add the lines to create both projects passing this new plugin as it is done. 

Also, we need to add the mock to the tests made with them, here: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/testdata/test.sh#L33-L45

(it will call make targets to verify the scafofld)